src/app/body/task-details/task-details.component.html

```diff
@@ -22,6 +22,28 @@ <h5 class="active nav-link navbar-text" id="head">Task Details</h5>
                         <li class=" active" *ngIf="(authService.afauth.user | async) && task.Status!=='Completed' && task.SprintNumber !== -2">
                             <span type="submit" class="btn material-icons text-light" data-bs-toggle="tooltip" data-bs-placement="top" data-toggle="modal" data-target="#exampleModal" title="Log work" (click)="logWorkPage()">pending_actions</span>
                         </li>
+                        <li class=" active pr-1 py-1 justify-content-between">
```

Comment:
The complete model can be broken and a new component can be designed for model, giving it power to reusability.

Reply:
okay will make another component and raise another PR.